### PR TITLE
ESSNTL-1579: Add mssql_version fact to system_profile

### PR DESCRIPTION
**Pull request by CodeHeeler**

### CODE REVIEW

**Comment by CodeHeeler** [comment]
Hi,
I tried to add our mssql_version field to puptoo for [ESSNTL-1579](https://issues.redhat.com/browse/ESSNTL-1579), but I do not have a system with mssql to test it with.  Hope this helps get this fact into our system_profile.  Let me know if someone has the bandwidth to help test this out.

Thanks!

File: src/puptoo/process/profile.py [hunk 1]
```diff
@@ -631,6 +635,18 @@ def _get_gpg_pubkey_packages(rpms):
...
+def _get_mssql_server_package(packages):
+    """
+    Get the mssql-server package from the latest packages from the
+    InstalledRpms parser.
+    """
+    result = None
+    for package in packages:
+        if package.name == "mssql-server":
+            result = package
+    return package
```
Comment (vkrizan) [inline src/puptoo/process/profile.py:647]: `s/package/result/`

File: src/puptoo/process/profile.py [hunk 2]
```diff
@@ -631,6 +635,18 @@ def _get_gpg_pubkey_packages(rpms):
...
 
+def _get_mssql_server_package(packages):
+    """
+    Get the mssql-server package from the latest packages from the
+    InstalledRpms parser.
+    """
+    result = None
+    for package in packages:
+        if package.name == "mssql-server":
+            result = package
```
Comment (vkrizan) [inline src/puptoo/process/profile.py:646]: add `break`

File: src/puptoo/process/profile.py (hunk 1 above)
Comment (CodeHeeler) [reply src/puptoo/process/profile.py]: thanks! this is what happens when I code late on a friday evidently :)

**Comment by r14chandra** [comment]
Thanks @CodeHeeler for creating the PR. Tested on EE. Getting no error logs. Hence, merging this PR for further testing on Stage env.

**Pull request merged**